packages/aws-cdk-lib/aws-lambda/lib/function.ts

```diff
@@ -295,6 +295,20 @@ export interface FunctionOptions extends EventInvokeConfigOptions {
    */
   readonly allowAllOutbound?: boolean;
```

Comment:
We should update the property documentation as this allows all traffic except IPv6.
It might not be very clear for the users.

Reply:
Updated. Thanks for the suggestion.